.circleci/config.yml

```diff
@@ -439,21 +451,66 @@ jobs:
           report_failure: << parameters.report_failure >>
           message: "metrics"
 
-  trigger-compatibility-verification:
-    docker:
-      - image: cimg/python:3.10.4
-    steps:
-      - checkout
-      - run:
-          name: Install python depedencies
-          command: |
-            pip3 install PyYAML
-            pip3 install GitPython
-            pip3 install requests
-      - run:
-          name: Trigger compatibility test
-          command: |
-            python3 scripts/ci-trigger/ci-compatibility-start-pipeline.py --target-slug mapbox/mapbox-sdk --token ${CIRCLECI_API_TOKEN_MAPBOX_MAPS_IOS_INTERNAL} --config mapbox-maps-ios=${CIRCLE_SHA1} --origin-slug mapbox/mapbox-maps-ios --platform ios
+  sdk-e2e-test:
+      docker:
+        - image: "mbxci/linux-base:ebee268fbe3d078cdacefee679f09c609006cdf4"
+      resource_class: small
+      parameters:
+          validation:
+              default: false
+              type: boolean
+      steps:
+          - add_ssh_keys:
+              fingerprints:
+                  - 3b:cd:47:bf:57:9c:e5:36:b0:4d:5f:12:5e:d3:b3:3e
+          - checkout
+          - run:
+              command: |
+                  curl -Ls https://mapbox-release-engineering.s3.amazonaws.com/mbx-ci/latest/mbx-ci-linux-amd64 > mbx-ci && chmod 755 ./mbx-ci
+                  ./mbx-ci aws setup
+              name: Install mbx-ci
```

Comment:
We have `install-mbx-ci` command that seem to be doing pretty much the same thing

Reply:
Done - switched to the existing command. I made couple small fixes to it to get it working.